packages/next-urql/examples/5-with-suspense-exchange/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "5-with-suspense-exchange",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "next dev",
+    "prestart": "cd ../../ && yarn build && cd examples/3-with-custom-exchange"
+  },
+  "dependencies": {
+    "@urql/exchange-suspense": "^1.8.3",
+    "graphql": "^14.5.8",
+    "graphql-tag": "^2.10.1",
+    "isomorphic-unfetch": "^3.0.0",
+    "next": "^9.2.0",
+    "wonka": "^4.0.7"
```

Comment:
Starting in v0.3.1 `next-urql` now ships with a direct `urql` dependency: https://github.com/FormidableLabs/urql/blob/master/packages/next-urql/package.json#L54 This should bring `wonka` along with it, so I'd expect this example _not_ to need to list `wonka` as a dep (although I'm certain `yarn` will deduplicate it). I think we ought to remove for clarity's sake.